Classes/model/API/AlertAPIModels.swift

```diff
@@ -46,3 +64,40 @@ extension API {
     public let routeIDs: [String]?
   }
 }
+
+extension API.Alert.Action: Codable {
+  private enum CodingKeys: String, CodingKey {
+    case text
+    case type
+    case excludedStopCodes
+  }
+  
+  enum CodingError: Error {
+    case unknownType(String)
+  }
+  
+  public init(from decoder: Decoder) throws {
+    let container = try decoder.container(keyedBy: CodingKeys.self)
+    text = try container.decode(String.self, forKey: .text)
+    
+    let type = try container.decode(String.self, forKey: .type)
+    switch type {
+    case "rerouteExcludingStops":
+      let excludedStops = try container.decode([String].self, forKey: .excludedStopCodes)
+      self.type = .reroute(excludedStops)
+    default:
+      throw CodingError.unknownType("Decoding Error: \(dump(container))")
+    }
+  }
+  
+  public func encode(to encoder: Encoder) throws {
+    var container = encoder.container(keyedBy: CodingKeys.self)
+    try container.encode(text, forKey: .text)
+    
+    switch type {
+    case .reroute(let excludedStops):
+      try container.encode("excludedStopCodes", forKey: .type)
```

Comment:
This doesn't match decode. Please us the same string for the type in both.

Reply:
Done.